chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/watcher/outgoing/ChaosMonkeyRestTemplateWatcherIntegrationTest.java

```diff
@@ -63,22 +63,23 @@ class ExceptionAssaultIntegrationTest {
 
         @Test
         public void testRestTemplateExceptionAssault() {
-            assertThatThrownBy(() -> this.demoRestTemplateService.callWithRestTemplate()).hasMessage("500 INTERNAL_SERVER_ERROR");
+            assertThatThrownBy(() -> demoRestTemplateService.callWithRestTemplate()).hasMessage("500 INTERNAL_SERVER_ERROR");
         }
     }
 
-    @Slf4j
     @SpringBootTest(properties = {"chaos.monkey.enabled=true", "chaos.monkey.watcher.rest-template=true", "chaos.monkey.assaults.latency-active=true",
             "chaos.monkey.test.rest-template.time-out=20"}, classes = {ChaosDemoApplication.class})
     @ActiveProfiles("chaos-monkey")
-    static class LatencyAssaultIntegrationTest {
+    @Nested
+    @Disabled
```

Comment:
I guess that was added by accident? However it seems that the Exception has changed from SSLException to io.netty.handler.timeout.ReadTimeoutException

EDIT: This seems to work: assertThatThrownBy(() -> demoRestTemplateService.callWithRestTemplate()).isInstanceOf(ReadTimeoutException.class);